docusaurus/docs/quickstarts/network/hosted.md

```diff
@@ -1,168 +1,235 @@
+commands and verify systemd has started the processes and see the status:
+
+```bash
+# run me
+sudo systemctl -q status ziti-controller --lines=0 --no-pager
+sudo systemctl -q status ziti-router --lines=0 --no-pager
+```
+
+```bash
+# example output
+$ sudo systemctl -q status ziti-controller --lines=0 --no-pager
+● ziti-controller.service - Ziti-Controller
+     Loaded: loaded (/etc/systemd/system/ziti-controller.service; disabled; vendor preset: enabled)
+     Active: active (running) since Thu 2021-11-11 19:05:46 UTC; 8s ago
+   Main PID: 2375 (ziti-controller)
+      Tasks: 7 (limit: 1154)
+     Memory: 43.7M
+     CGroup: /system.slice/ziti-controller.service
+             └─2375 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-controller run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/co…
+
+$ sudo systemctl -q status ziti-router --lines=0 --no-pager
+● ziti-router.service - Ziti-Router for ip-10-0-0-1-edge-router
+     Loaded: loaded (/etc/systemd/system/ziti-router.service; disabled; vendor preset: enabled)
+     Active: active (running) since Thu 2021-11-11 19:05:47 UTC; 8s ago
+   Main PID: 2385 (ziti-router)
+      Tasks: 6 (limit: 1154)
+     Memory: 231.4M
+     CGroup: /system.slice/ziti-router.service
+             └─2385 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-router run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ip-10…
+```
+
+## Adding Environment Variables Back to the Shell
+
+If you log out and log back in again you can source the '.env' file located at:
+`~/.ziti/quickstart/$(hostname -s)/$(hostname -s).env`.
+
+```bash
+# run me
+source ~/.ziti/quickstart/$(hostname -s)/$(hostname -s).env
+```
+
+```bash
+# example output
+$ source ~/.ziti/quickstart/$(hostname -s)/$(hostname -s).env
+adding /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.20.2 to the path
+
+$ echo $ZITI_HOME
+/home/ubuntu/.ziti/quickstart/ip-10-0-0-1
+```
+
+## Next Steps
+
+- [Use the Overlay](#use-the-overlay)
+- [Install Ziti Admin Console (ZAC)](#install-ziti-admin-console-zac)
+<!-- - Add a Private Router -->
+- [Add a Second Public Router](#add-a-second-public-router)
+- [Change Admin Password](#change-admin-password)
+- [Delete everything and start over](#delete-everything-and-start-over)
+
+### Use the Overlay
+
+Now you have your zero trust overlay network in place, you probably want to try it out. Head on over to
+[the services quickstart](../services/index.md) and start the journey to understanding how to use OpenZiti.
+
+### Install Ziti Admin Console (ZAC)
+
+This is an optional server app and web console for Ziti network administration.
+
+[Installation guide](../zac/installation.md)
+
+### Add a Second Public Router
+
+Routers automatically form a mesh of router links when there's more than one. In order for your express-installed router to participate in the mesh it needs a firewall exception to expose its "link listener" on port `10080/tcp`, the default port.
```

Comment:
> Routers automatically form a mesh of router links when there's more than one.

this is not factually true and is a complication that doesn't need to be on this page. there's no need for exposition here. i thought we were going to go with a "next steps" idea for down here?

Reply:
This is in the "next steps" section.

> In order for multiple routers to form transit links, they need a firewall exception to expose the "link listener" port. The default port is `10080/tcp`.